### PR TITLE
Increase label length to 255 characters rather than 100 characters.

### PR DESCRIPTION
**Pull request by Xon**
Found the 100 characters a little short, lets more than double it.

As it is varchar it doesn't make a difference unless we want to add an index to it (and have to deal with unicode).

### CODE REVIEW

File: upload/library/Sidane/Threadmarks/Install.php [hunk 1]
```diff
@@ -111,4 +103,60 @@ public static function uninstall()
...
     $db->delete('xf_permission_entry_content', "permission_id = 'sidane_tm_view'");
   }
+
+  public static function modifyColumn($table, $column, $oldDefinition, $definition)
+  {
+    $db = XenForo_Application::get('db');
+    $hasColumn = false;
+    if (empty($oldDefinition))
+    {
+      $hasColumn = !$db->fetchRow('SHOW COLUMNS FROM `'.$table.'` WHERE Field = ?', $column);
```
Comment (sidane) [inline upload/library/Sidane/Threadmarks/Install.php]: The return value of `fetchRow` shouldn't be inverted here as `$hasColumn` will be `false` when the column exists and the column will not be modified.

When I upgraded the add-on with this change, `label` remained as a `varchar(100)`.

File: upload/library/Sidane/Threadmarks/Install.php [hunk 2]
```diff
@@ -111,4 +103,60 @@ public static function uninstall()
...
+    }
+    else
+    {
+      $hasColumn = !$db->fetchRow('SHOW COLUMNS FROM `'.$table.'` WHERE Field = ? and Type = ?', array($column,$oldDefinition));
```
Comment (sidane) [inline upload/library/Sidane/Threadmarks/Install.php]: Same as above, shouldn't be inverted.

**Comment by Xon** [comment]
I can't believe I missed that. I'm going to need to stand up a fresh copy of XF and install the older versions and then test to make sure the install process is working properly.

**Comment by Xon** [comment]
Rebased, and Fixed installer helpers.

**Comment by sidane** [comment]
:+1:

**Pull request merged**